src/jgikbase/idmapping/core/user.py

```diff
@@ -5,12 +5,12 @@
 from jgikbase.idmapping.util.util import not_none
 
 
-class Authsource:
+class AuthsourceID:
     """
-    An authorization source for a user.
+    An identifier for an authorization source for a user.
```

Comment:
Suggested doc here:
"An authentication source is any server, program, or agent that can give you back a plaintext username when it is given a secret token."